src/parser/parser.py

```diff
@@ -0,0 +1,64 @@
+"""Parses a PDF document.
+"""
+import logging
+
+
+import slate
+
+
+logging.basicConfig(level=logging.INFO)
+
+
+class Article:
+    """Class for storing articles.
+    """
+
+    def __init__(self, number, text):
+        self.number = number
+        self.text = text
+
+
+def count_articles(pdf_text):
+    """Identifies articles and returns a list of Article objects.
+
+    Args:
```

Comment:
What counts as an Article? There's more than one article per PDF?

Reply:
Went ahead and added the specifications to the comments